devops/db/liquibase/changelogs/payment-transaction-gateway/master-changelog.xml

```diff
@@ -9,4 +9,5 @@
   <include file="./incremental-changelogs/V2_PM_895-changelog.xml" relativeToChangelogFile="true"></include>
   <include file="./incremental-changelogs/V3_PM_913-changelog.xml" relativeToChangelogFile="true"></include>
   <include file="./incremental-changelogs/V4_PM_917-changelog.xml" relativeToChangelogFile="true"></include>
+  <include file="./incremental-changelogs/V5_PM_988-changelog.xml" relativeToChangelogFile="true"></include>
```

Comment:
@andrearovere always remember to add your incremental-changelog file to the master-changelog, otherwise your scripts will not be executed